### ISSUE TITLE
Create release policy document

### ISSUE THREAD

**Issue by mmulji-ic** [cosmos/gaia#1286]
## Summary

Alignment with other teams
Don’t do releases at EOD
2 body policy
Monday - Wednesday
On-call roll
Someone has reduced work capacity during the 2 days afterwards

## Proposal

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by yaruwangway** [comment]
hi @mmulji-ic, the release policy doc is different from release process right? release process is another issue here https://github.com/cosmos/gaia/pull/1271

**Comment by mmulji-ic** [comment]
Add:
- tests automated for new features
- feature freeze
- code freeze
- 1 wk testing without issue otherwise reset clock
- comms once testing is done
- align with eng. council
- Maintain live release calendar

**Comment by mmulji-ic** [comment]
@mmulji-ic update hub-eng and send to eng-council to review.

**Comment by yaruwangway** [comment]
@mmulji-ic can we close this one ?

**Comment by mpoke** [comment]
@mmulji-ic What's the latest status with this issue?

**Comment by mpoke** [comment]
Closing in favor of https://github.com/cosmos/gaia/issues/2357
